### ISSUE TITLE
fabric-1.20.1-1.4.5 stops modpack from loading, requesting backport

### ISSUE THREAD

**Issue by jwrichards1982** [henkelmax/sound-physics-remastered#246]
### Bug description

tldr I download a modpack and spend two hours hunting down the mod that keeps the modpack form loading.
I download the latest fabric-1.21.1-1.4.6 and drop it into a set of mods I am working on for my own modpack and it loads fine.
No crash logs are produced when it happens, I am using:
inux kernel 6.10.6-arch1-1 (64-bit), KDE Plasma 6.1.5 (framework 6.5.0, Qt 6.7.2) Wayland graphics platform, JRE-OpenJDK 21, mesa-1:24.1.6-1 amdgpu for graphics and pipewire sound serveruesting backport

### Minecraft version

1.20.1

### Mod version

1.20.1-1.4.5

### Mod loader and version

Fabric 0.16.0

### Steps to reproduce

I Install Aged modpack v3.0.2 for minecraft 1.20.1 on modrinth, start the instance, loading progress almost gets to the end and the instance just stops. No crash logs and latest.log is complete greek to me.
I install any other modpack for minecraft 1.21.1 and sound phsyics is no longer a problem.
I try any other modpack for minecraft 1.20.1 and drop sound phsyics in that one and get the same behavior.
results of tessting;
1.20.1 modpacks, minecraft stops loading on the loading screen till process ends on its own.
1.21.1 modpacks, minecraft works fine.
1.20.1 JUST sound physics, minecraft stops loading on the loading screen till process ends on its own.
1.21.1 JUST sound physics, minecraft works fine.
Looks like whatever changed between fabric-1.20.1-1.4.5 and fabric-1.21.1-1.4.6 something broke.

### Expected behavior

I expect Minecraft to continue loading, not stop loading

### Log files

https://gist.github.com/jwrichards1982/488b5d3daa800adec7ec4cfb647e8761

### Videos

https://youtu.be/y2k34EMDGlM

**Comment by henkelmax** [comment]
There is no version 1.4.6 for Minecraft 1.20.1.

**Comment by jwrichards1982** [comment]
I know there is no 1.4.6 for 1.20.1. There is 1.4.5 which is broken.
1.4.5 for 1.20.1 is broken. 1.4.6 for 1.21.1 is working. I was hoping to request a step back a 1.4.5.5 I guess, but your link worked a charm. Thank you! I swear I searched and didn't find anything. I could have been on the wrong project. I am juggling a few things at one time on my computer. Thank you!

**Comment by henkelmax** [comment]
No problem :)
